### ISSUE TITLE
Colab

### ISSUE THREAD

**Issue by pmiclea** [michhar/pytorch-mask-rcnn-samples#2]
Hi,
I try to use your example in Google colab, and I get a issue about an error in  setup.py egg_info by running requirements. CAn you help me on that. Here the error:

error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
 error: metadata-generation-failed

**Comment by Yash-Wasalwar-07** [comment]
Same here!, I also got the same error... Please help us out !!

**Comment by michhar** [comment]
Hi Folks.  I apologize, but I'm not familiar with the Python environment in Colab, so without more info it is a little challenging to advise.  Maybe you can share how you are trying to set things up?  What the environment is like?

**Comment by Yash-Wasalwar-07** [comment]
Yes sure, what I am doing is taking the Setup.ipynb file from your repo, and before running it in colab, activating the hardware accelerator of colab by changing it to GPU. Then running your code line by line, then the above kind of error occurs. I will also include the screenshot of the code below...
<img width="950" alt="Screenshot 2023-04-04 at 10 17 27" src="https://user-images.githubusercontent.com/71366827/229689619-a7ea55cb-181c-4882-9941-2aa50f7f8fef.png">
<img width="883" alt="Screenshot 2023-04-04 at 10 17 42" src="https://user-images.githubusercontent.com/71366827/229689668-94b20cb9-1f32-415a-8d34-f7e20380dc3f.png">

**Comment by michhar** [comment]
Suggestion 1:
- Try removing the version in `requirements.txt` associated with `cmake` and see if it's just a version clash from python 3.9 to that `cmake`

If that doesn't work, suggestion 2:
- Use or create a new environment (conda or venv if you can in colab) that is Python 3.6 which was used in the project

If environments get messy, I have a docker container you may use as well that also starts Jupyterhub, but haven't tried in over 4 years much on this project so hopefully one of these suggestions above, or containers, can help.  Let me know.
- CPU:  https://hub.docker.com/r/rheartpython/cvdeep
- GPU:  https://hub.docker.com/r/rheartpython/cvdeep_gpu
